Provider.php

```diff
@@ -45,11 +43,13 @@ public function getLastTouched(array $tableNamesOrIds)
 
         foreach ($tableNamesOrIds as $t) {
             if ($t == '*') {
-                $timestamp = $this->connection->fetchColumn(
-                    'SELECT UNIX_TIMESTAMP(MAX(last_touched)) FROM wfd_meta'
-                );
+                $lastTouchOnAnyTable = $this->connection->fetchAssoc('SELECT MAX(last_touched) lastTouchedString FROM wfd_meta');
+                if ($lastTouchOnAnyTable['lastTouchedString'] === null) {
```

Comment:
If we could get rid of the (unnecessary) array, that would be extra great. Maybe `fetchCol` helps? 

Otherwise, leave it as-is.